docs/Embedded/EnkluVolume.md

```diff
@@ -0,0 +1,18 @@
+---
+id: EnkluVolume
+title: Embedding an Experience
+---
+
+_\[Note: Access to this feature requires a [Studio Subscription](https://enklu.com/pricing). [Contact us](/contact) for details.\]_
+
+To embed a scene in your Unity project, add an EnkluVolume prefab to your scene. Find it by searching for "EnkluVolume" in the project tab and ensuring the "In Packages" option is selected. Once the prefab is in your scene, select it and log in to your [Enklu Cloud account](https://cloud.enklu.com) in the inspector tab.
```

Comment:
This might be unnecessary, but just to clear up any ambiguity, it might be good to change "embed a scene" to something like "embed an Enklu Cloud project" since there are two "scenes" in the same sentence. Though I doubt most people will be confused about which scene refers to what, defining it more explicitly could be helpful.